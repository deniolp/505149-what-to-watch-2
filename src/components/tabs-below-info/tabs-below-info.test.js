import React from 'react';
import renderer from 'react-test-renderer';

import TabsBelowInfo from './tabs-below-info';
import filmsListMock from '../../mocks/films';

describe(`TabsBelowInfo`, () => {
  const film = filmsListMock[0];

  it(`renders correctly`, () => {
    const tree = renderer.create(
        <TabsBelowInfo
          film={film}
          label={`Overview`}
        />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
