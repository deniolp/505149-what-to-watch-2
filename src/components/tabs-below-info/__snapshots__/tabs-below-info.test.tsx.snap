// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabsBelowInfo renders correctly 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      8.3
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Very good
      </span>
      <span
        className="movie-rating__count"
      >
        187
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      Deserunt ex esse tempor aliqua labore ex ipsum ipsum. Minim elit anim consectetur eu cillum. Quis ut ut minim anim sit irure anim non officia id ex. Elit in ea ad elit consequat magna culpa excepteur Lorem. Excepteur sit sit enim irure enim in amet excepteur. Sunt ullamco et cupidatat est adipisicing occaecat nostrud voluptate aliquip consequat commodo adipisicing. Ipsum exercitation est consectetur aute magna anim dolore id enim.
    </p>
    <p>
          Sunt ullamco et cupidatat est adipisicing occaecat nostrud voluptate aliquip consequat commodo adipisicing. Ipsum exercitation est consectetur aute magna anim dolore id enim.
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        Christopher Nolan
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        Bill Murray, Jack Nicholson, Edward Norton, Jude Law, Dustin Hoffman
         and other
      </strong>
    </p>
  </div>,
]
`;
